src/main/java/org/example/c02controlflow/generate/MultiplicationTableOfAGivenInteger.java

```diff
@@ -1,16 +1,14 @@
 package org.example.c02controlflow.generate;
 
-import java.util.Scanner;
 
 public class MultiplicationTableOfAGivenInteger {
-    public static void main(String[] args) {
-        int multipliedNumber;
-        Scanner sc = new Scanner(System.in);
-        System.out.print("Enter integer:");
-        int number = sc.nextInt();
+    public static int[] displayMultiplicationTable(int number) {
```

Comment:
This should return a string pattern like:
5 x 1 = 5
5 x 2 = 10
5 x 3 = 15
.
.
.

Reply:
Efectuată modificarea